cpp/lib/src/ZoteroHarvesterConversion.cc

```diff
@@ -1089,7 +1107,9 @@ void ConversionTasklet::run(const ConversionParams &parameters, ConversionResult
             if (new_metadata_record.url_.empty())
                 throw std::runtime_error("no URL set");
 
-            if (ExcludeOnlineFirstRecord(new_metadata_record, parameters)) {
+            if (ExcludeUnsesiredItemTypes(new_metadata_record)) {
+                ++result->num_skipped_since_undesired_item_type_;
```

Comment:
Please do not use blocks for one-liners.

Reply:
you're right, but the problem is that the `continue` statement is missing